examples/codes/run_process_example.ex.py

```diff
@@ -0,0 +1,376 @@
+        "shldoth": 0.8,
+        "ddwex": 0.15,
+        "gapomin": 0.2,
+        # Vertical build inputs
+        "d_vv_top": 0.3,
+        "vgap2": 0.05,
+        "shldtth": 0.3,
+        "divfix": 0.621,
+        "d_vv_bot": 0.3,
+        # HCD inputs
+        "pinjalw": 51.0,
+        "gamma_ecrh": 0.3,
+        "etaech": 0.4,
+        "bscfmax": 0.99,
+        # BOP inputs
+        "etath": 0.375,
+        "etahtp": 0.87,
+        "etaiso": 0.9,
+        "vfshld": 0.6,
+        "tdwell": 0.0,
+        "tramp": 500.0,
+        # CS / PF coil inputs
+        "t_crack_vertical": 0.4e-3,
+        "fcuohsu": 0.7,
+        "ohhghf": 0.9,
+        "rpf2": -1.825,
+        "cptdin": [4.22e4, 4.22e4, 4.22e4, 4.22e4, 4.3e4, 4.3e4, 4.3e4, 4.3e4],
+        "ipfloc": [2, 2, 3, 3],
+        "ncls": [1, 1, 2, 2],
+        "ngrp": 4,
+        "rjconpf": [1.1e7, 1.1e7, 6.0e6, 6.0e6, 8.0e6, 8.0e6, 8.0e6, 8.0e6],
+        # TF coil inputs
+        "n_tf": 16,
+        "casthi": 0.06,
+        "casths": 0.05,
+        "ripmax": 0.6,
+        "dhecoil": 0.01,
+        "tftmp": 4.75,
+        "thicndut": 2.0e-3,
+        "tinstf": 0.008,
+        # "tfinsgap": 0.01,
+        "tmargmin": 1.5,
+        "vftf": 0.3,
+    }
+)
+
+# %%[markdown]
+# PROCESS has many different models with integer-value 'switches'. We can specify
+# these choices as follows:
+
+# %%
+for model_choice in (
+    BootstrapCurrentScalingLaw.SAUTER,
+    ConfinementTimeScalingLaw.IPB98_Y2_H_MODE,
+    PlasmaCurrentScalingLaw.ITER_REVISED,
+    PlasmaProfileModel.CONSISTENT,
+    PlasmaPedestalModel.PEDESTAL_GW,
+    PlasmaNullConfigurationModel.SINGLE_NULL,
+    EPEDScalingModel.SAARELMA,
+    BetaLimitModel.THERMAL,
+    DensityLimitModel.GREENWALD,
+    AlphaPressureModel.WARD,
+    PlasmaGeometryModel.CREATE_A_M_S,
+    PowerFlowModel.SIMPLE,
+    ShieldThermalHeatUse.LOW_GRADE_HEAT,
+    SecondaryCycleModel.INPUT,
+    CurrentDriveEfficiencyModel.ECRH_UI_GAM,
+    OperationModel.PULSED,
+    PFSuperconductorModel.NBTI,
+    SolenoidSwitchModel.SOLENOID,
+    CSSuperconductorModel.NB3SN_WST,
+    TFSuperconductorModel.NB3SN_WST,
+    TFWindingPackTurnModel.INTEGER_TURN,
+    FISPACTSwitchModel.OFF,
+    PrimaryPumpingModel.PRESSURE_DROP_INPUT,
+    TFNuclearHeatingModel.INPUT,
+    CostModel.TETRA_1990,
+    AvailabilityModel.INPUT,
+    OutputCostsSwitch.NO,
+):
+    template_builder.set_model(model_choice)
+
+# %%[markdown]
+# Some of these model choices also require certain input values
+# to be specified. If these are not specified by the user, default
+# values are used, which may not be desirable. Let us see what
+# we're still missing:
+
+# %%
+inputs = template_builder.make_inputs()
+
+# %%[markdown]
+# And now let's add those missing inputs:
+
+# %%
+template_builder.add_input_value("qnuc", 1.3e4)
+template_builder.add_input_value("n_layer", 20)
+template_builder.add_input_value("n_pancake", 20)
+
+
+# %%[markdown]
+# Finally, let us run PROCESS with our inputs:
+
```

Comment:
Could we add a sentence here about why the first argument is `{}` and link to eg:

```python
#  [External code example](../external_code.ex.py)
```